twitter_ads/http.py

```diff
@@ -139,19 +134,6 @@ def __init__(self, code, headers, **kwargs):
         except ValueError:
             self._body = raw_response_body
 
-        if 'x-rate-limit-reset' in headers:
-            self._rate_limit = int(headers['x-rate-limit-limit'])
-            self._rate_limit_remaining = int(headers['x-rate-limit-remaining'])
-            self._rate_limit_reset = datetime.fromtimestamp(int(headers['x-rate-limit-reset']))
```

Comment:
I'm fine with not converting raw header values to a specific format like this. We can just leave it as is so that users can process as they want to.